api/go.mod

```diff
@@ -3,6 +3,7 @@ module kubevirt.io/ssp-operator/api
 go 1.18
 
 require (
+	github.com/openshift/api v0.0.0
```

Comment:
I'm sorry for confusion, but I found out that you can simply `require` here the needed api version.
I thought there was some problem with modules so we needed to use this hack with `require` `v0.0.0` and then `replace` the needed version.

Can you change this line to:
```
github.com/openshift/api v0.0.0-20220124143425-d74727069f6f // release-4.10
```

Reply:
when I did that (same for other comments) it actually failed go mod tidy :/